Library/Formula/vmtouch.rb

```diff
@@ -0,0 +1,15 @@
+class Vmtouch < Formula
+  desc "Portable file system cache diagnostics and control"
+  homepage "http://hoytech.com/vmtouch"
+  url "https://github.com/hoytech/vmtouch/archive/vmtouch-1.0.2.tar.gz"
+  sha256 "48d41e5b805a0fcac46c12feeb4650e4b9def44b75d528d7c66c2f8b3a747c39"
+  head "https://github.com/hoytech/vmtouch.git"
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}", "MANDIR=#{man}"
+  end
+
+  test do
+    system "#{bin}/vmtouch", "#{bin}/vmtouch"
```

Comment:
In its simplest form, vmtouch reports the cached pages of a file. The test
passes the vmtouch binary as the subject file. If the call doesn't error
out, we seem to have a working install.

On Saturday, 5 December 2015, Dominyk Tiller <notifications@github.com>
wrote:

> In Library/Formula/vmtouch.rb
> <https://github.com/Homebrew/homebrew/pull/46677#discussion_r46761987>:
>
> > @@ -0,0 +1,15 @@
> > +class Vmtouch < Formula
> > +  desc "Portable file system cache diagnostics and control"
> > +  homepage "http://hoytech.com/vmtouch"
> > +  url "https://github.com/hoytech/vmtouch/archive/vmtouch-1.0.2.tar.gz"
> > +  sha256 "48d41e5b805a0fcac46c12feeb4650e4b9def44b75d528d7c66c2f8b3a747c39"
> > +  head "https://github.com/hoytech/vmtouch.git"
> > +
> > +  def install
> > +    system "make", "install", "PREFIX=#{prefix}", "MANDIR=#{man}"
> > +  end
> > +
> > +  test do
> > +    system "#{bin}/vmtouch", "#{bin}/vmtouch"
>
> What's the test doing here, out of curiosity?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/46677/files#r46761987>.
>